debian/changelog

```diff
@@ -1,3 +1,26 @@
+cloud-init (24.1.3-0ubuntu2) noble; urgency=medium
+
+  * Upstream bug fix release based on 24.1.4.
```

Comment:
Think this will raise eyebrows given the upstream version is based on 24.1.3?

Reply:
I think we are good here as we are saying it's `Upstream bug fix release` not `Upstream snapshot release`. The difference being that we are only pulling in bug fixes, not any features from 24.1.4